pkg/reconciler/v1alpha1/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -706,8 +723,11 @@ func TestResolvePipelineRun_ResourceBindingsDontExist(t *testing.T) {
 				ObjectMeta: metav1.ObjectMeta{
 					Name: "pipelinerun",
 				},
+				Status: v1alpha1.PipelineRunStatus{
+					TaskRunNames: map[string]string{},
+				},
 			}
-			_, err := ResolvePipelineRun(pr, getTask, getClusterTask, getResource, tt.p.Spec.Tasks, providedResources)
+			_, _, err := ResolvePipelineRun(pr, getTask, getClusterTask, getResource, tt.p.Spec.Tasks, providedResources)
```

Comment:
if we keep the names in the `ResolvePipelineRun` interface, we'd want to be asserting against them in at least some of our unit tests

(but it will be easier if we can move that logic into its own function that can be tested separately :D)

Reply:
Oh, but we do! Twice in fact! =)